.gitlab-ci.yml

```diff
@@ -763,6 +763,31 @@ library:ci-mczify:
   - build:edge+flambda
   - library:ci-mathcomp
 
+library:ci-mathcomp_base:
+  extends: .ci-template
+
+library:ci-finmap:
+  extends: .ci-template
+  needs:
+  - build:base
+  - library:ci-mathcomp_base
+
+library:ci-bigenough:
+  extends: .ci-template
+  needs:
+  - build:base
+  - library:ci-mathcomp_base
+
+library:ci-analysis:
+  extends: .ci-template
+  needs:
+  - build:base
+  - library:ci-mathcomp_base
+  - library:ci-finmap
+  - library:ci-bigenough
+  - plugin:ci-elpi  # for Hierarchy Builder
+  # we need a mathcomp_base here as elpi requires OCaml < 4.13
```

Comment:
@gares any chance of getting a elpi compatible with new ocaml?

Reply:
the problem is camlp5. how long can we wait? I'm replacing it with menhir, but it will take some time

Reply:
> the problem is camlp5

(insert here PTSD flash images)

More seriously, since we have our minimal parsing engine in Coq derived from camlp5, would it be faster to simply port it to the gramlib API?

Reply:
nope, I will kill it for real this time.
I have a branch for recent camlp5, but that version requires perl to build, so it only builds fine on linux. We can use it temporarily if necessary.

Reply:
no hurry here